features/sprite-clones.js

```diff
@@ -9,21 +9,21 @@ if (
       document.querySelector(
         "span.scratchtoolsSpriteCloneCounter"
       ).textContent = `${(
-        ScratchTools.Scratch.vm.editingTarget.sprite.clones.length - 1
+        ScratchTools.Scratch.vm.editingTarget?.sprite.clones.length - 1
       ).toString()} clones`;
     } else {
       var foundIt = false;
       document.querySelectorAll("div").forEach(function (el) {
         if (foundIt === false) {
-          if (el.className.startsWith("sprite-info_row_")) {
+          if (el.className.startsWith("sprite-info_row_1om5V") &&! el.className.includes("sprite-info_row-primary_10JrS")) {
```

Comment:
Why did you change the selector for this one?

Reply:
It's divided like that:
![image](https://user-images.githubusercontent.com/63302372/233859041-f2f8c6c2-1345-4ed9-a829-28e6b9762d25.png)
![image](https://user-images.githubusercontent.com/63302372/233859170-53a6cf1a-e67f-4390-9c4b-6a8eddb5ae55.png)
And the old script was inserting the clone counter after the first box, I changed it to insert the button after the second box, so that it wouldn't break in half